tests/logging_tests/tests.py

```diff
@@ -105,6 +106,47 @@ def test_django_logger_debug(self):
             self.logger.debug('debug')
             self.assertEqual(self.logger_output.getvalue(), '')
 
+    def test_page_found_no_warning(self):
+        with self.settings(DEBUG=True):
```

Comment:
Good idea. I've fixed this. I had gotten this style by imitating several other functions already present in the same file. So I fixed this in a subsequent commit that fixes all of them, (rather than just amending my one commit).